integration_tests/test_numpy_04.py

```diff
@@ -0,0 +1,14 @@
+from ltypes import f64
+from numpy import array
+
+def test_array():
+    x: list[f64] = array([1.0, 2.0, 3.0])
```

Comment:
```suggestion
    x: f64[3] = array([1.0, 2.0, 3.0])
```

Reply:
If we change, I get:
```console
$ lpython integration_tests/test_numpy_04.py
semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> integration_tests/test_numpy_04.py:5:5
  |
5 |     x: f64[3] = array([1.0, 2.0, 3.0])
  |     ^           ^^^^^^^^^^^^^^^^^^^^^^ type mismatch ('f64[4]' and 'list[f64]')
```
how do I differentiate between `list` and `array`

Reply:
A list is a type `List_t`, while an array is a type (like an `Integer_t` or `Real_t`) with dimensions set.

Reply:
Got it, Thank you!